components/footer/home/homeBtn.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./homeBtn.module.css";
+
+/* import homeImg from "./homeBtn.stories"; */
+
+export type HomeBtnProps = {
+  src: string;
+};
+
+function HomeBtn({ ...props }: HomeBtnProps) {
```

Comment:
Actually, this is not a button, but it's called `HomeBtn`.

Reply:
True, it is a link. It is confusing, I am going to rename it. Thanks